packages/adapter/addon/index.ts

```diff
@@ -55,7 +55,7 @@ import EmberObject from '@ember/object';
   @class Adapter
   @extends EmberObject
 */
-export default EmberObject.extend({
+export default class Adapter extends EmberObject {
```

Comment:
Does it have to extend EmberObject now?

Reply:
I bet imagine for many users, this would be an unnoticeable change.  However, technically this would be a breaking change.